app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,13 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @posts = Post.where(author_id: params[:user_id])
+    @comments = Comment.all
+    @users = User.all
```

Comment:
[OPTIONAL] There is no need for the instance variables `@comments` and `@users`. It would be nicer to remove them to keep your code clean.